config/services.yml

```diff
@@ -0,0 +1,49 @@
+---
+services:
+
+  _defaults:
+    autoconfigure: true
+    autowire: true
+    bind:
+      $cloudConfigFilepath: '@=service("Acquia\\Cli\\Helpers\\LocalMachineHelper").getCloudConfigFilepath()'
+      # $acliConfigFilepath: '/home/dane/.acquia/acquia-cli.json'
+      Webmozart\KeyValueStore\JsonFileStore $datastoreCloud: '@datastore.cloud'
+      # Webmozart\KeyValueStore\JsonFileStore $datastoreAcli: '@datastore.acli'
+    public: true
+
+  Acquia\Cli\:
+    exclude: ../src/Kernel.php
+    public: false
+    resource: ../src
+
+  AcquiaCloudApi\Connector\Client: ~
+
+  AcquiaCloudApi\Connector\ConnectorInterface: ~
+
+  Psr\Log\LoggerInterface:
+    alias: Symfony\Component\Console\Logger\ConsoleLogger
+
+  Symfony\Component\Console\Application: ~
+
+  Symfony\Component\Console\Input\ArgvInput: ~
+
+  Symfony\Component\Console\Input\InputInterface:
+    alias: Symfony\Component\Console\Input\ArgvInput
+
+  Symfony\Component\Console\Logger\ConsoleLogger: ~
+
+  Symfony\Component\Console\Output\ConsoleOutput: ~
```

Comment:
Can we add some comments to explain what's happening on some of these lines?

Reply:
These are all just 'services' (specific classes) that are part of some constructor in our application. Symfony probably _could_ autodiscover all of these, but it's way more performant (at least for vendor-provided services) to enumerating, that's what we're doing here.

I think we should only comment on ones that might be non-obvious, anything in particular bother you?

Reply:
I added some comments.